lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -56,6 +56,11 @@ def execute?(command, input = nil)
         $CHILD_STATUS.success?
       end
 
+      def rpm_greater_than_version?(name, target_version)
```

Comment:
nitpick: this method is checking >= not just > so maybe `rpm_gte_version` ?

Reply:
i'm good with that!

Reply:
updated the method name :+1:

Reply:
still works - ship it 👍